_episodes/02-setup.md

```diff
@@ -16,7 +16,6 @@ we need to configure a few things. Below are a few examples
 of configurations we will set as we get started with Git:
 
 *   our name and email address,
-*   to colorize our output,
 *   what our preferred text editor is,
 *   and that we want to use these settings globally (i.e. for every project)
```

Comment:
Sweet, @raynamharris ! Now that you are at it, could you please add the missing period at the end of this sentence?